src/lib/evil/unposix/stdlib.h

```diff
@@ -3,7 +3,7 @@
 
 #include "unimplemented.h"
 
-#include <evil_stdlib.h>
+#include_next <stdlib.h>
```

Comment:
What this has to do with wrap dependency?

Reply:
More with fixing how linkage/including was done. Actually this is going to change again when we finish #40.